ui/cryptomaterial/segmented_control.go

```diff
@@ -75,7 +77,15 @@ func (sc *SegmentedControl) SetEnableSwipe(enable bool) {
 	sc.isSwipeActionEnabled = enable
 }
 
-func (sc *SegmentedControl) Layout(gtx C, body func(gtx C) D) D {
+// Layout handles the segmented control's layout, it receives an optional isMobileView bool
+// parameter which is used to determine if the segmented control should be displayed in mobile view
+// or not. If the parameter is not provided, isMobileView defaults to false.
+func (sc *SegmentedControl) Layout(gtx C, body func(gtx C) D, isMobileView ...bool) D {
+	sc.isMobileView = false
+	if len(isMobileView) > 0 {
+		sc.isMobileView = isMobileView[0]
+	}
+
```

Comment:
Errm, can we consider `sc.isMobileView = len(isMobileView) > 0  && isMobileView[0]`?